src/updater.cpp

```diff
@@ -92,7 +94,68 @@ void Updater::checkForUpdates()
     key = "windows";
 #endif
     QString link = links[key].toString();
+#if __APPLE__
```

Comment:
this is good, we could later have a settings in the GUI settings to disable update notifications and or provide an option  to auto update without even prompting or asking user (for smooth auto-update experiecne)